### ISSUE TITLE
[FEATURE REQ] Cost Management APIs - or workaround in the meantime

### ISSUE THREAD

**Issue by kierenj** [Azure/azure-sdk-for-net#9451]
I see the folder is there for the CostManagement APIs, but no classes yet. I was wondering where it sat on the roadmap?

In the meantime, I'm very keen to access this. I'm using the fluent API to do all kinds of things, and I'd like to be able to craft a REST request to the Cost Management endpoint(s), even if manually.

I've been wading through the various service/manager base classes, but haven't yet circled in on how I can simply grab whatever auth headers I need to issue a REST request.

To put it another way - I have an `IAzure`. How can I use the classes in `Microsoft.Azure.Management.Fluent` to grab a token for CostManagement and issue REST requests?

I managed to build a `RestClient`, but if I use the `RootHttpHandler` (I think that's what its called) in a new `HttpClient`, I do get `Unauthorized`, so I'm sure there's more to it.

**Comment by ramaganesan-rg** [comment]
Thanks for your feedback. we will evaluate against the other priorities. At this point, we dont have plans for .NET SDK. 
Feel free to use REST APIs using AAD login.
